pkgs/top-level/all-packages.nix

```diff
@@ -19677,14 +19677,21 @@ with pkgs;
 
   tesh = callPackage ../tools/text/tesh {};
 
-  texinfo413 = callPackage ../development/tools/misc/texinfo/4.13a.nix { };
-  texinfo4 = texinfo413;
-  texinfo5 = callPackage ../development/tools/misc/texinfo/5.2.nix { };
-  texinfo6_5 = callPackage ../development/tools/misc/texinfo/6.5.nix { }; # needed for allegro
-  texinfo6_7 = callPackage ../development/tools/misc/texinfo/6.7.nix { }; # needed for gpm, iksemel and fwknop
-  texinfo6 = callPackage ../development/tools/misc/texinfo/6.8.nix { };
-  texinfo7 = callPackage ../development/tools/misc/texinfo/7.0.nix { };
+  texinfo-versions = callPackages ../development/tools/misc/texinfo { };
```

Comment:
Name it `texinfoPackages` instead.

Reply:
Personally I find this a bit misleading in the sense that `texinfoPackages` might mean _additional_ packages _for_ texinfo (similar to `python3Packages` or `nodePackages`) rather than different version _of_ texinfo.
How could this ambiguity be avoided?

Reply:
Well, you have a point with there being a difference, but why does this difference matter? `firefoxPackages`, for example, is a set of `firefox` versions. Sure, I would not be against using `texinfoVersions` for this (but not `texinfo-versions` or some such since https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax rules apply, and thus it should be `texinfoVersions`, since it is not a package). But why make the distinction?

Reply:
Thinking about this some more I think texinfoPackages is fine as AFAIK texinfo does not offer third-party packages like TeX, Node, or Python does. I'll rework this accordingly :)